packages/form-js-editor/src/features/properties-panel/entries/TextEntry.js

```diff
@@ -25,17 +24,6 @@ export default function TextEntry(props) {
     }
   ];
 
-  // todo: skipped to make the release without too much risk
-  // if (templating.isTemplate(field.text)) {
-  //   entries.push(simpleBoolEntryFactory({
-  //     id: 'strict',
```

Comment:
@Skaiir I deleted these comments, should I create an issue to replace them?

Reply:
Nah we've been good without it, if it's needed it'll come up again.